### PR TITLE
Proposals/audit filing status

### PR DESCRIPTION
**Pull request by JieXiong9119**
See proposal
Implementation in schema added

### CODE REVIEW

File: BuildingSync.xsd [hunk 1]
```diff
@@ -5929,6 +5929,17 @@
...
         </xs:complexType>
       </xs:element>
+      <xs:element name="AuditFilingStatus" minOccurs="0">
+        <xs:annotation>
+          <xs:documentation>The status of audit filing.</xs:documentation>
+        </xs:annotation>
+        <xs:simpleType>
+          <xs:restriction base="xs:string">
+            <xs:enumeration value="Initial filing"/>
+            <xs:enumeration value="Amended Filing"/>
```
Comment (macintoshpie) [inline BuildingSync.xsd:5939]: Not sure what our convention is, (@nllong ?) but should probably be consistent with capitalization

Comment (JieXiong9119) [reply BuildingSync.xsd:5939]: > Not sure what our convention is, (@nllong ?) but should probably be consistent with capitalization

Thanks for catching it! I think it should be "Amended filing". Correct me if I'm wrong @nllong .

File: proposals/2021/Add AuditFilingStatus.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+# Add AuditFilingStatus
+
+## Overview
+
+This proposal
+
+## Justification
+
+This field is only specific to Atlanta reports, but could be a useful input for others if requested.
```
Comment (corymosiman12) [inline proposals/2021/Add AuditFilingStatus.md:9]: hey Jie - I think just clarify with this justification section that this originally came about in working directly with the city of Atlanta, but we believe it is a good field and therefore are adding it to the schema...something like that.

File: BuildingSync.xsd [hunk 3]
```diff
@@ -5929,6 +5929,17 @@
           </xs:sequence>
         </xs:complexType>
       </xs:element>
+      <xs:element name="AuditFilingStatus" minOccurs="0">
+        <xs:annotation>
+          <xs:documentation>The status of audit filing.</xs:documentation>
```
Comment (corymosiman12) [inline BuildingSync.xsd:5934]: Maybe rephrase: `The status of an audit filing, used to clarify whether or not this audit report is an initial submission (Initial filing) or an amendment to a previously submitted report (Amended filing).`

**Review by corymosiman12 — CHANGES_REQUESTED** [review]
Minor changes requested. I think changing to `Amended filing` is the correct approach.

File: proposals/2021/Add AuditFilingStatus.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
+# Add AuditFilingStatus
+
+## Overview
+
+This proposal is to add the `AuditFilingStatus` element as a child of `ReportType` element, and to restrict the options with `Initial filing` and `Amended filing`.
```
Comment (JieXiong9119) [inline proposals/2021/Add AuditFilingStatus.md:5]: Is it under `ReportType` element or `Report` element instead? @corymosiman12

**Review by yan130 — APPROVED** [review]

File: proposals/2021/Add AuditFilingStatus.md (hunk 4 above)
Comment (corymosiman12) [reply proposals/2021/Add AuditFilingStatus.md:5]: image

@JieXiong9119 you would end up putting it under the 'ReportType' element, which gets implemented in the 'Report' element. Make sense?

Comment (corymosiman12) [reply proposals/2021/Add AuditFilingStatus.md:5]: Which you have already done, my b. You are good.

File: BuildingSync.xsd (hunk 1 above)
Comment (nllong) [reply BuildingSync.xsd]: yes, `Amended filing` is the correct casing. For most cases we are using sentence case

**Review by nllong — APPROVED** [review]
